### PR TITLE
System.Net changes for README.md

### PR DESCRIPTION
**Pull request by davidsh**
- Update System.Net.Security description
- Add System.Net.Sockets description

### CODE REVIEW

**Comment by davidsh** [comment]
cc: @CIPop @SidharthNabar @pgavlin @stephentoub

File: README.md [hunk 1]
```diff
@@ -93,7 +93,8 @@ More libraries are coming soon (the overall list of items we currently plan to m
...
 |<sub>**System.Net.Requests**<br/>[![MyGet Package](https://img.shields.io/myget/dotnet-core/v/System.Net.Requests.svg)](https://www.myget.org/gallery/dotnet-core)</sub>|<sub>Provides older classes (such as HttpWebRequest and HttpWebResponse) for sending HTTP requests and receiving HTTP responses from a resource identified by a URI. _This library is available primarily for compatibility; developers should prefer the classes in the System.Net.Http package._</sub>|
-|<sub>**System.Net.Security**<br/>[![MyGet Package](https://img.shields.io/myget/dotnet-core/v/System.Net.Security.svg)](https://www.myget.org/gallery/dotnet-core)</sub>|<sub>Provides types, such as System.Net.Security.SslStream, that enable secure network communication between applications and hosts.</sub>|
+|<sub>**System.Net.Security**<br/>[![MyGet Package](https://img.shields.io/myget/dotnet-core/v/System.Net.Security.svg)](https://www.myget.org/gallery/dotnet-core)</sub>|<sub>Provides types, such as System.Net.Security.SslStream, that uses SSL/TLS protocols to provide secure network communication between client and server endpoints.</sub>|
```
Comment (CIPop) [inline README.md]: Do we need to update Packages\descriptions.json as well?

**Comment by CIPop** [comment]
LGTM

File: README.md (hunk 1 above)
Comment (davidsh) [inline README.md]: >Do we need to update Packages\descriptions.json as well?

That change will be done in TFS separately.

**Comment by SidharthNabar** [comment]
LGTM

**Comment by stephentoub** [comment]
LGTM

**Pull request merged**